internal/service/controlplaneipaddresses/data_source_test.go

```diff
@@ -0,0 +1,29 @@
+package controlplaneipaddresses_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+func TestAccControlPlaneIpAddressesDS_basic(t *testing.T) {
+	dataSourceName := "data.mongodbatlas_control_plane_ip_addresses.test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		Steps: []resource.TestStep{
+			{
+				Config: configBasic,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(dataSourceName, "outbound.aws.us-east-1.0"),
```

Comment:
Worth using `resource.TestCheckResourceAttrWith(dataSourceName, "outbound.aws.us-east-1.0", acc.MatchesExpression(...))`?

Reply:
Included a `CIDRBlockExpression` check, thanks for suggestion